src/app/pages/settings/commentSettings.js

```diff
@@ -1,137 +1,150 @@
 	};
 
 	const disableCommentsNoticeText = () => {
 		return disableCommentsOldPosts
-			? __('Comments on old posts are disabled.', 'wp-plugin-bluehost')
-			: __('Comments are allowed on old posts.', 'wp-plugin-bluehost');
+			? __( 'Comments on old posts are disabled.', 'wp-plugin-bluehost' )
+			: __( 'Comments are allowed on old posts.', 'wp-plugin-bluehost' );
 	};
 
 	const toggleDisableCommentsOldPosts = () => {
-		bluehostSettingsApiFetch({ disableCommentsOldPosts: !disableCommentsOldPosts }, setError, (response) => {
-			setDisableCommentsOldPosts(!disableCommentsOldPosts);
-		});
+		bluehostSettingsApiFetch(
+			{ disableCommentsOldPosts: ! disableCommentsOldPosts },
+			setError,
+			// eslint-disable-next-line no-unused-vars
+			( response ) => {
+				setDisableCommentsOldPosts( ! disableCommentsOldPosts );
+			}
+		);
 	};
 
 	const notifySuccess = () => {
-		notify.push("disable-old-posts-comments-notice", {
+		notify.push( 'disable-old-posts-comments-notice', {
 			title: disableCommentsNoticeTitle(),
-			description: (
-				<span>
-					{disableCommentsNoticeText()}
-				</span>
-			),
-			variant: "success",
+			description: <span>{ disableCommentsNoticeText() }</span>,
+			variant: 'success',
 			autoDismiss: 5000,
-		});
+		} );
 	};
 
-	useUpdateEffect(() => {
-		setStore({
+	useUpdateEffect( () => {
+		setStore( {
 			...store,
 			disableCommentsOldPosts,
-		});
+		} );
 
 		notifySuccess();
-	}, [disableCommentsOldPosts]);
+	}, [ disableCommentsOldPosts ] );
 
 	return (
 		<ToggleField
 			id="disable-comments-toggle"
-			label={__(
+			label={ __(
 				'Disable comments for older posts',
 				'wp-plugin-bluehost'
-			)}
-			checked={disableCommentsOldPosts}
-			onChange={toggleDisableCommentsOldPosts}
+			) }
+			checked={ disableCommentsOldPosts }
+			onChange={ toggleDisableCommentsOldPosts }
 			className="nfd-mb-2"
 		/>
 	);
-}
+};
 
-const CloseCommentsDays = ({ setError, notify }) => {
-	const { store, setStore } = useContext(AppStore);
-	const [closeCommentsDays, setNumCloseCommentsDays] = useState(
+const CloseCommentsDays = ( { setError, notify } ) => {
+	const { store, setStore } = useContext( AppStore );
+	const [ closeCommentsDays, setNumCloseCommentsDays ] = useState(
 		store.closeCommentsDays
 	);
 
 	const closeCommentsDaysNoticeTitle = () => {
-		return (
-			__('Comments setting saved ', 'wp-plugin-bluehost')
-		);
+		return __( 'Comments setting saved ', 'wp-plugin-bluehost' );
 	};
 
 	const closeCommentsDaysNoticeText = () => {
 		//`Comments on posts are disabled after ${closeCommentsDays} days.`
 		return (
-			__('Comments on posts are disabled after ', 'wp-plugin-bluehost') +
+			__(
+				'Comments on posts are disabled after ',
+				'wp-plugin-bluehost'
+			) +
 			closeCommentsDays +
-			_n(' day.', ' days.', parseInt(closeCommentsDays), 'wp-plugin-bluehost')
+			_n(
+				' day.',
+				' days.',
+				parseInt( closeCommentsDays ),
+				'wp-plugin-bluehost'
```

Comment:
This should be using `sprintf` to fill the `closeCommentsDays` and the `day`/`days`, not all languages will follow the same grammatical format as English.